src/components/InvertedFlatList/BaseInvertedFlatList/index.tsx

```diff
@@ -19,11 +19,12 @@ function defaultKeyExtractor<T>(item: T | {key: string} | {id: string}, index: n
     return String(index);
 }
 
-type BaseInvertedFlatListProps<T> = Omit<FlatListProps<T>, 'data' | 'renderItem' | 'initialScrollIndex'> & {
+type BaseInvertedFlatListProps<T> = Omit<FlatListProps<T>, 'data' | 'renderItem' | 'initialScrollIndex' | 'CellRendererComponent'> & {
     shouldEnableAutoScrollToTopThreshold?: boolean;
     data: T[];
     renderItem: ListRenderItem<T>;
     initialScrollKey?: string | null;
+    CellRendererComponent?: React.FC<PropsWithChildren>;
```

Comment:
Can we just use the original type of `CellRendererComponent` instead of `React.FC` here?